templates/forum/thread.haml

```diff
@@ -18,7 +18,9 @@
       {% message_partial message user 0 %}
 
   .reply-block
-    %textarea.form-control
-    .flex
-      .flex-spacer
-      %button.btn.btn-primary.btn-lg{:type => "submit"} Reply
\ No newline at end of file
+    %form{:action =>'' , :method => 'post'} {% csrf_token %}
```

Comment:
Action needs to be /forum/thread/id

Reply:
I was not able to add the id part. However, if the action remains empty, then the request will be on the same url, which is what we want here.

Reply:
Didn't know that! 👍